linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_unlock_time/bash/shared.sh

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_sle,multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ bash_instantiate_variables("var_accounts_passwords_pam_tally2_unlock_time") }}}
+# Use a non-number regexp to force update of the value of the deny option
+{{{ bash_ensure_pam_module_options('/etc/pam.d/common-auth', 'auth', 'required', 'pam_tally2.so', 'unlock_time', '${var_accounts_passwords_pam_tally2_unlock_time}', "${var_accounts_passwords_pam_tally2_unlock_time}") }}}
+
+{{{ bash_ensure_pam_module_options('/etc/pam.d/common-account', 'account', 'required', 'pam_tally2.so', '', '', '') }}}
```

Comment:
I recommend to use the `bash_ensure_pam_module_option` macro instead of the `bash_ensure_pam_module_options` (with s). More details in the previous comment.